app/src/main/java/com/example/pix/home/fragments/HomeFragment.java

```diff
@@ -79,13 +86,19 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         colors.add(Color.GREEN);
 
 
-        ImageView profile = view.findViewById(R.id.home_profile_icon);
-        TextView pix = view.findViewById(R.id.tv_score);
-        ViewPager viewPager = view.findViewById(R.id.vpPager);
+        profile = view.findViewById(R.id.home_profile_icon);
+        pix = view.findViewById(R.id.tv_score);
+        viewPager = view.findViewById(R.id.vpPager);
```

Comment:
Made these global so that when we return from the ProfileFragment the alphas and tints are corrected.